src/snowflake/snowpark/relational_grouped_dataframe.py

```diff
@@ -178,23 +178,26 @@ def __init__(
     def _to_df(
         self,
         agg_exprs: List[Expression],
+        exclude_grouping_columns: bool = False,
```

Comment:
should we add the option to select certain grouping columns if needed? right now, it select all, or deselect all.

Reply:
per our offline discussion, make it private as it is only added for SAS